src/Pages/yeonseonglee/Login/Login.js

```diff
@@ -0,0 +1,170 @@
+import React, { Component } from "react";
+import { withRouter } from "react-router-dom";
+import "./Login.scss";
+
+class Login extends Component {
+  componentDidMount() {
+    const name = document.getElementsByClassName("idPwdForm")[0];
+    const password = document.getElementsByClassName("idPwdForm")[1];
+    const button = document.getElementsByClassName("btn")[0];
+    const login = document.getElementsByClassName("login")[0];
+
+    function activate() {
+      if (name.value.length > 0 && password.value.length > 0) {
+        button.style.backgroundColor = "#0095F6";
+      } else {
+        button.style.backgroundColor = "#C1DFFD";
+      }
+    }
+    login.addEventListener("keyup", activate);
+  }
+
+  goToMain = () => {
+    this.props.history.push("/main");
+  };
+
+  render() {
+    return (
+      <div class="login">
+        <main className="wrapper">
+          <div className="wrapper-left">
+            <img
+              className="leftimg"
+              src={
+                process.env.PUBLIC_URL + "/images/yeonseonglee/instagram123.png"
+              }
+              alt="insta"
+            />
+          </div>
+          <div className="wrapper-right">
+            <article className="artitop">
+              <div className="logo">
+                <img
+                  src={
+                    process.env.PUBLIC_URL + "images/yeonseonglee/logo_text.png"
+                  }
+                  alt="logo"
+                />
+              </div>
+              <div className="login">
+                <input
+                  className="idPwdForm"
+                  type="text"
+                  placeholder="전화번호, 사용자이름 또는 이메일"
+                />
+                <input
+                  className="idPwdForm"
+                  type="password"
+                  placeholder="비밀번호"
+                />
```

Comment:
여기서 onChange 함수를 props로 사용할 수 있습니다!
```suggestion
                <input
                  onChange={this.handleId}
                  className="idPwdForm"
                  type="text"
                  placeholder="전화번호, 사용자이름 또는 이메일"
                />
               
```

Reply:
props 사용해서 수정했습니다.